gocd/totestmanager.gocd.yaml.erb

```diff
@@ -7,11 +7,7 @@ pipelines:
       openSUSE:Factory:PowerPC
       openSUSE:Factory:RISCV
       openSUSE:Factory:zSystems
-      openSUSE:Leap:15.5:Images
-      openSUSE:Leap:15.5:ARM
-      openSUSE:Leap:15.5:ARM:Images
       openSUSE:Leap:15.6:Images
-      openSUSE:Leap:15.6:ARM
```

Comment:
Out of curiosity, was this also intentional?

Reply:
Yes. 15.6 was released a long time ago so those projects are no longer used.